src/rust/en.mangakatana/src/helper.rs

```diff
@@ -0,0 +1,184 @@
+use aidoku::{
+	prelude::format,
+	std::html::Node,
+	std::{String, Vec},
+	MangaContentRating, MangaStatus, MangaViewer,
+};
+
+/// Percent-encode any non-ASCII characters in a string.
+pub fn urlencode(string: String) -> String {
```

Comment:
isn't this `encode_uri` in aidoku-rs helpers?

Reply:
Didn't know that was a thing. I just copied `urlencode` from another extension's helper.